R/stanreg.R

```diff
@@ -73,6 +89,11 @@ stanreg <- function(object) {
     residuals <- ytmp - mu
   }
   names(eta) <- names(mu) <- names(residuals) <- ynames
+  if (family$family == "beta") {
```

Comment:
actually if it's gonna be checked this many times might as well just do `is_betareg <- is.beta(family$family)` once towards the top and then just check `is_betareg` when you need to.